### PR TITLE
fix(docker): fixes #853 added restart policy

### LINKED ISSUE #853: Add a restart policy to docker compose services
## Description
 Chaos Genius services do not restart if the docker daemon or the system restart due to any reason.
 A few scenarios where this could occur
 - Underlying hardware is degraded so AWS would ask us to stop and start instance.
 - When you want to change instance type you need to stop server and start again. Here we don't need to ssh to start containers.
 - It's possible due to some reason container got killed because we are running multiple containers in same machine so it would start again.
 ## Describe the solution you'd like
 Add `restart: unless-stopped` to all the CG services in all docker compose files. 
 Reference: https://docs.docker.com/compose/compose-file/compose-file-v3/#restart.

### PR DESCRIPTION
**Pull request by varunp2k**
closes #853

### CODE REVIEW

File: docker-compose.fluentd.yml [hunk 1]
```diff
@@ -107,7 +110,6 @@ services:
       timeout: 45s
       interval: 10s
       retries: 10
-    restart: always
```
Comment (Samyak2) [inline docker-compose.fluentd.yml:110]: `chaosgenius-db` should also be `restart: unless-stopped`

**Review by Samyak2 — CHANGES_REQUESTED** [review]

**Comment by varunp2k** [comment]
@Samyak2 Please check

File: docker-compose.latest.yml [hunk 2]
```diff
@@ -85,7 +87,7 @@ services:
   chaosgenius-db:
     container_name: chaosgenius-db
     image: postgres:13
-    restart: always
+    restart: unless-stopped
     healthcheck:
       test: [ "CMD", "pg_isready", "-q", "-d", "chaosgenius_data", "-U", "postgres" ]
       timeout: 45s
```
Comment (Samyak2) [inline docker-compose.latest.yml:93]: This one still has a `restart: always` on line 94

**Review by Samyak2 — CHANGES_REQUESTED** [review]
Also, we should update the stable `docker-compose.yml` and `docker-compose.thirdparty.yml` too.

File: docker-compose.latest.yml (hunk 2 above)
Comment (varunp2k) [reply docker-compose.latest.yml:93]: @Samyak2 imo, we should update the docker-compose during the release time.

**Review by Samyak2 — APPROVED** [review]
LGTM

**Pull request merged**